suites/tipset/miner_flow.go

```diff
@@ -42,7 +42,7 @@ func TestMinerCreateProveCommitAndMissPoStChallengeWindow(t *testing.T, factory
 	minerWorker, minerWorkerID := td.NewAccountActor(address.BLS, abi_spec.NewTokenAmount(1_000_000_000))
 	// The address of the miner actor
 	minerActorID := utils.NewIDAddr(t, utils.IdFromAddress(minerWorkerID)+1)
-	createMinerRet := td.ComputeInitActorExecReturn(builtin_spec.StoragePowerActorAddr, 0, minerActorID)
```

Comment:
pretty sure this was just wrong, but i might be missing something.

Reply:
Since a storage miner actor is [created by](https://github.com/filecoin-project/specs-actors/blob/3f5510bf61303d1f629f8fce26ebe3194c911f93/actors/builtin/miner/miner_actor.go#L69-L70) the storage power actor I believe the StoragePowerActorAddr is the correct parameter here.